### ISSUE TITLE
Incorrect instructions on finding tracker. 

### ISSUE THREAD

**Issue by gibsop1** [seemoo-lab/AirGuard#84]
There is a typo in the instructions on what to do if you're being tracked. It says "do not move to a safe location before deactivating" I presume this should say "move to a safe location"

**Comment by DominikNovosel** [comment]
No, I'm pretty sure it's correct. Assuming "safe location" means something like your home, school, workplace or other location where you may regularly reside, it would be illogical and dangerous to go to any of these before deactivating the tracker and making sure no-one can follow you.
